### PR TITLE
fix: update test and coverage

### PR DESCRIPTION
**Pull request by klu909**
## Context

we recently enabled user to set test/coverage url in build detail page

## Objective

update test/coverage documentation to reflect the change

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/user-guide/metadata.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ jobs:
       - set-coverage-and-test-results: |
           meta set tests.coverage 100 # this should be the coverage percentage number
           meta set tests.results 10/10 # this should be `pass_tests_number/total_tests_number`
+          meta set tests.results /test/testReport.html # this should be a relative path to a build artifact
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
          meta set tests.resultsUrl /test/testReport.html # this should be a relative path to a build artifact
```

File: docs/user-guide/metadata.md [hunk 2]
```diff
@@ -182,6 +182,8 @@ jobs:
...
+          meta set tests.results /test/coverageReport.html # this should be a relative path to a build artifact
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
          meta set tests.coverageUrl /test/coverageReport.html # this should be a relative path to a build artifact
```

File: docs/user-guide/metadata.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ These settings will result in Git checks that look like:
 
 ### Coverage and Test Results
 
-You can populate coverage and test results on build page from a Screwdriver build through using metadata. Screwdriver UI will read from `tests.coverage` and `tests.results` in metadata and display them.
+You can populate coverage and test results and set their url to build artifact on build page from a Screwdriver build through using metadata. Screwdriver UI will read from `tests.coverage`, `tests.results`, `tests.coverageUrl` and `tests.resultsUrl` in metadata and display/set them accordingly.
```
Comment (tkyi) [inline docs/user-guide/metadata.md]:
```suggestion
You can populate coverage results and test results, along with their respective URLs, on the build page by setting metadata during a Screwdriver build. The Screwdriver UI will read from `tests.coverage`, `tests.results`, `tests.coverageUrl`, and `tests.resultsUrl` metadata and display/set them accordingly.
```

**Comment by tkyi** [comment]
Might be good to add a screenshot of the build page where these links and data are displayed.

**Comment by klu909** [comment]
> Might be good to add a screenshot of the build page where these links and data are displayed.

original doc already included an image

File: docs/user-guide/metadata.md (hunk 2 above)
Comment (klu909) [reply docs/user-guide/metadata.md]: changed

**Review by tkyi — APPROVED** [review]

**Pull request merged**